app/src/main/res/layout/screen_access_denied.xml

```diff
@@ -8,53 +8,79 @@
   tools:ignore="Overdraw">
 
   <LinearLayout
-    android:id="@+id/applock_logo_container"
+    android:id="@+id/outerContainer"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:background="?colorPrimary"
-    android:orientation="vertical"
+    android:orientation="horizontal"
+    android:gravity="center_vertical"
     android:paddingBottom="@dimen/spacing_16"
     android:paddingEnd="@dimen/spacing_24"
     android:paddingStart="@dimen/spacing_24"
-    android:paddingTop="@dimen/spacing_4">
+    android:paddingTop="@dimen/spacing_64">
 
-    <Button
-      android:id="@+id/applock_logout"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:background="@drawable/background_rounded_clearbutton"
-      android:foreground="?attr/selectableItemBackgroundBorderless"
-      android:minHeight="0dp"
-      android:minWidth="0dp"
-      android:paddingBottom="2dp"
-      android:paddingEnd="@dimen/spacing_8"
-      android:paddingStart="@dimen/spacing_8"
-      android:paddingTop="2dp"
-      android:text="@string/applock_logout"
-      android:textAppearance="@style/Clinic.V2.TextAppearance.Button2.White100"
-      android:visibility="gone"
-      tools:ignore="UnusedAttribute" />
+      <TextView
```

Comment:
The spacing between this widget and the cardview does not seem to match the designs (https://zpl.io/bep4zXl). Please check?